automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -53,24 +59,21 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
           withWorkspace(projectName, "AuthDomainSpec_share", Set(authDomainName), List(AclEntry(Config.Users.george.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
             withCleanUp {
               withSignIn(Config.Users.george) { listPage =>
-                val summaryPage = listPage.openWorkspaceDetails(projectName, workspaceName)
+                val summaryPage = listPage.enterWorkspace(projectName, workspaceName)
```

Comment:
i like this pattern, but it's not used everywhere (e.g. PublishSpec). we should remember to update later. and definitely add to the docs.